src/charm.py

```diff
@@ -1406,6 +1407,12 @@ def _reconciliation(self, _event: EventBase) -> None:
             self.unit.status = WaitingStatus("Waiting for storage")
             _event.defer()
             return
+        # Ensure the data directory for WordPress is writable for the WordPress user.
+        self._container().exec(
+            f"chown {self._WORDPRESS_USER}:{self._WORDPRESS_GROUP} -R {self._WP_UPLOADS_PATH}".split(
+                " "
```

Comment:
nit: calling `.split()` without an argument is similar to `.split(" ")`, or we can just provide a list to `exec` instead of splitting the argument string as the argument list is not very long.

Reply:
Nice, changed.